packages/frontend-design-poc/src/components/FilterBar/FilterBar.tsx

```diff
@@ -134,6 +139,20 @@ export const FilterBar = ({ onFilterChange, fields, initialFilters = [] }: Filte
     [activeFilters, onFilterChange],
   );
 
+  const handleSaveSearch = () => {
+    const searchHistory: SavedSearch[] = getSearchHistory();
+    const newSearch: SavedSearch = {
+      id: Date.now(),
```

Comment:
id kunne f.eks. vært en hash code av søk + aktive filtre, så er det lettere å sjekke om den er lagret senere bare ved å sjekke hash?

Reply:
Id her vil vel bli generert av databasen? Satte den til timestamp kun for testing, før persistering backend er på plass.